spec/Tests.spec.js

```diff
@@ -19,4 +19,18 @@ describe("Parse Server example", () => {
       expect(e.message).toBe('Saving test objects is not available.');
     }
   })
+  it('coverage for /', async () => {
+    const {text, headers} = await Parse.Cloud.httpRequest({
+      url: 'http://localhost:30001/'
+    })
+    expect(headers['content-type']).toContain('text/html');
+    expect(text).toBe('I dream of being a website.  Please star the parse-server repo on GitHub!');
+  })
+  it('coverage for /test', async () => {
+    const {text, headers} = await Parse.Cloud.httpRequest({
+      url: 'http://localhost:30001/test'
+    })
+    expect(headers['content-type']).toContain('text/html');
+    expect(text).toContain('<title>Parse Server Example</title>');
+  })
```

Comment:
Doesn't lint require a new line at EOF? If not, maybe we should use the same lint config as Parse Server does, so developers don't necessarily jump back and forth between two styles of code?

Reply:
I didn't run lint on spec files as I get errors such as:

```
'describe' is not defined
'it' is not defined  
```

Would you have any suggestions as to how to improve this?

Reply:
See https://github.com/parse-community/parse-server/blob/master/spec/.eslintrc.json

Reply:
Lint should run for spec and src

Reply:
Thanks for the pointers!